ui/admin/app/templates/scopes/scope/credential-stores/credential-store/-actions.hbs

```diff
@@ -0,0 +1,17 @@
+<Rose::Dropdown
+  @text={{t 'actions.manage'}}
+  @dropdownRight={{true}}
+  as |dropdown|
+>
+  <dropdown.link @route='scopes.scope.credential-stores.new' @model={{@model}}>
```

Comment:
I know this route is not correct since it does not keep the organization on the URL. I will work on this later, probably will push the change to this same PR.